content/docs/api-reference/integrations/integrations/get.mdx

```diff
@@ -18,4 +15,6 @@ _openapi:
 
 {/* This file was generated by Fumadocs. Do not edit this file directly. Any changes should be made by running the generation command again. */}
 
+Return all the integrations the user has created for that organization. Review v.0.17.0 changelog for a breaking change
```

Comment:
Shall we add v0.17.0 changelog link?

Reply:
Unfortuent;y this is auto generatedby open api, so it will be a bit tricky